src/dotnet/Streaming.UI.Blazor/Components/AudioRecorder/constants.ts

```diff
@@ -0,0 +1,5 @@
+export const SAMPLES_PER_MS = 16;
+export const SAMPLES_PER_WINDOW_32 = SAMPLES_PER_MS * 32;
+export const SAMPLES_PER_WINDOW_30 = SAMPLES_PER_MS * 30;
+export const SAMPLE_RATE = 16000;
```

Comment:
Why did sample rate become lower? It was 48000 Hz, wasn't it?

Reply:
Microphone recording sample rate is 16K, but playback is 48K for Android and iOS by default, so when there is only one AudioContext mnicrophone stream has to be resampled, Now we use two different AudioContexts